http_opa/util/utils_test.go

```diff
@@ -0,0 +1,66 @@
+package util
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/infobloxopen/atlas-authz-middleware/v2/http_opa/exception"
+)
+
+func TestGetBearerFromRequest(t *testing.T) {
+	tests := []struct {
+		name           string
+		request        *http.Request
+		expectedBearer string
+		expectedError  error
+	}{
+		{
+			name: "Bearer token is present",
+			request: &http.Request{
+				Header: http.Header{
+					"Authorization": []string{"Bearer token"},
+				},
+			},
+			expectedBearer: "token",
+		},
+		{
+			name: "Bearer token is not present",
+			request: &http.Request{
+				Header: http.Header{
+					"Authorization": []string{"Basic token"},
+				},
+			},
+			expectedError: exception.ErrAbstrAuthHeaderMissing,
+		},
+		{
+			name: "Bearer token is empty",
+			request: &http.Request{
+				Header: http.Header{
+					"Authorization": []string{""},
+				},
+			},
+			expectedError: exception.ErrAbstrAuthHeaderMissing,
+		},
+		{
+			name: "Bearer token is present along with other headers",
+			request: &http.Request{
+				Header: http.Header{
+					"Authorization": []string{"Basic basic.token", "Bearer bearer.token"},
+				},
+			},
+			expectedBearer: "bearer.token",
```

Comment:
add test case for 
```
Header: http.Header{
	"Authorization": []string{"Basic basic.token", "Bearer "},
},
```
and handle changes accordingly.

Reply:
based on the slack chat, we decided not to do this (no need to return error in case of "Bearer "). since this behaviour is consistent with earlier implementation. and this function is not concerned about the returned value being invalid jwt string (like empty string or 123) that is checked by another jwt validation method after fetching the token from this function.